src/phpDocumentor/Descriptor/Interfaces/ChildInterface.php

```diff
@@ -24,14 +23,14 @@ interface ChildInterface
     /**
      * Returns the parent for this descriptor.
      *
-     * @return DescriptorAbstract|Fqsen|string|null
+     * @return ElementInterface|Fqsen|string|null
```

Comment:
is string here an option?

Reply:
Should be investigated in a subsequent PR, I have copied this from existing code and dared not change it